assets/airports/cyhz.json

```diff
@@ -0,0 +1,2633 @@
+                ["aal", 1],
+                ["aca", 29],
+                ["dal", 1],
+                ["emb", 3],
+                ["fdx", 1],
+                ["fle", 2],
+                ["n/cessna", 8],
+                ["poe", 5],
+                ["pvl", 1],
+                ["ual", 1],
+                ["wen", 5],
+                ["wja", 15]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "CYHZ",
+            "category": "arrival",
+            "route": "EBONY.FUNDY2.CYHZ",
+            "altitude": [18000, 24000],
+            "speed": 250,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["aal", 1],
+                ["aca", 29],
+                ["dal", 1],
+                ["emb", 3],
+                ["fdx", 1],
+                ["fle", 2],
+                ["n/cessna", 8],
+                ["poe", 5],
+                ["pvl", 1],
+                ["ual", 1],
+                ["wen", 5],
+                ["wja", 15]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "CYHZ",
+            "category": "arrival",
+            "route": "ALLEX.FUNDY2.CYHZ",
+            "altitude": [18000, 24000],
+            "speed": 250,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["aal", 1],
+                ["aca", 29],
+                ["dal", 1],
+                ["emb", 3],
+                ["fdx", 1],
+                ["fle", 2],
+                ["n/cessna", 8],
+                ["poe", 5],
+                ["pvl", 1],
+                ["ual", 1],
+                ["wen", 5],
+                ["wja", 15]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "CYHZ",
+            "category": "arrival",
+            "route": "AGMIR.LIRLA2.CYHZ",
+            "altitude": 12000,
+            "speed": 280,
+            "method": "random",
+            "rate": 2,
+            "airlines": [
+                ["aca", 29],
+                ["cfg", 2],
+                ["fdx", 1],
+                ["fpo", 2],
+                ["kal", 2],
+                ["qtr", 2],
+                ["wja", 15]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "CYHZ",
+            "category": "arrival",
+            "route": "TUSKY.PEGGY2.CYHZ",
+            "altitude": [21000, 24000],
+            "speed": 250,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["aal", 1],
+                ["aca", 29],
+                ["dal", 1],
+                ["emb", 3],
+                ["fdx", 1],
+                ["fle", 2],
+                ["n/cessna", 8],
+                ["poe", 5],
+                ["pvl", 1],
+                ["ual", 1],
+                ["wen", 5],
+                ["wja", 15]
+            ]
+        }
+    ],
+    "maps": {
```

Comment:
Did you want to make any changes to the video map or are you leaving it as is?

Reply:
@Fechulo Yeah I did, strange it didn't push...